docs/changelog.rst

```diff
@@ -5,6 +5,11 @@ CHANGELOG
 2.94.0+dev (XXXX-XX-XX)
 -----------------------
 
+**New feature**
+
+- Handle very high resolution images (HD Views) that will automatically be tiled, for ``Trek``, ``POI`` and ``Site``
+- Handle annotations on HD Views (points, lines, polygons and text)
```

Comment:
You can now reference the related issue (https://github.com/GeotrekCE/Geotrek-admin/issues/3378)